service/src/test/kotlin/app/cash/backfila/client/FakeBackfilaClientServiceClient.kt

```diff
@@ -91,4 +93,7 @@ class FakeBackfilaClientServiceClient @Inject constructor() : BackfilaClientServ
     runBatchRequests.send(request)
     return runBatchResponses.receive().getOrThrow()
   }
+
+  override suspend fun finalizeBacfkill(request: FinalizeBackfillRequest) =
```

Comment:
We should do something similar to the Prepare and channel the requests. Then we can use this to test the service.